specs/Media.md

```diff
@@ -0,0 +1,183 @@
+    {
+        bool isAudioPlaying = webView.CoreWebView2.IsAudioPlaying;
+        bool isMuted = webView.CoreWebView2.IsMuted;
+        string currentDocumentTitle = webView.CoreWebView2.DocumentTitle;
+        if (is_audio_playing)
+        {
+            if (isMuted)
+            {
+                this.Title = "🔇 " + currentDocumentTitle;
+            }
+            else
+            {
+                this.Title = "🔊 " + currentDocumentTitle;
+            }
+        }
+        else
+        {
+            this.Title = currentDocumentTitle;
+        }
+    }
+    
+    void MuteCmdExecuted(object target, ExecutedRoutedEventArgs e)
+    {
+        webView.CoreWebView2.Mute();
+    }
+
+    void UnmuteCmdExecuted(object target, ExecutedRoutedEventArgs e)
+    {
+        webView.CoreWebView2.Unmute();
+    }
+```
+
+# API Notes
+
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+
+## Win32 C++
+
+```IDL
+interface ICoreWebView2StagingIsAudioPlayingChangedEventHandler;
+
+[uuid(71c906d9-4a4d-4dbe-aa1b-db64f4de594e), object, pointer_default(unique)]
+interface ICoreWebView2_2 : ICoreWebView2 {
+  /// Mutes all audio output from this CoreWebView2.
+  ///
+  /// \snippet AudioComponent.cpp Mute
+  HRESULT Mute();
+
+  /// Unmutes all audio output from this CoreWebView2.
+  ///
+  /// \snippet AudioComponent.cpp Unmute
+  HRESULT Unmute();
+
+  /// Indicates whether all audio output from this CoreWebView2 is muted or not.
+  ///
+  /// \snippet AudioComponent_Staging.cpp IsMuted
+  [propget] HRESULT IsMuted([out, retval] BOOL* value);
+  
+  /// Adds an event handler for the `IsAudioPlayingChanged` event.
+  /// `IsAudioPlayingChanged` runs when the audible state changes.
+  ///
+  /// \snippet AudioComponent.cpp IsAudioPlayingChanged
+  HRESULT add_IsAudioPlayingChanged(
+      [in] ICoreWebView2StagingIsAudioPlayingChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with `add_IsAudioPlayingChanged`.
+  HRESULT remove_IsAudioPlayingChanged(
+      [in] EventRegistrationToken token);
+  
+  /// Indicates whether any audio output from this CoreWebView2 is audible.
+  /// CoreWebView2 can be audible when muted, IsAudioPlaying is used to indicate
+  /// if there are audio currently playing.
+  ///
+  /// \snippet AudioComponent.cpp IsAudioPlaying
+  [propget] HRESULT IsAudioPlaying([out, retval] BOOL* value);
+}
+
+/// Implements the interface to receive `IsAudioPlayingChanged` events.  Use the
+/// IsAudioPlaying method to get the audible state.
+[uuid(5DEF109A-2F4B-49FA-B7F6-11C39E513328), object, pointer_default(unique)]
+interface ICoreWebView2StagingIsAudioPlayingChangedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.  No event args exist
+  /// and the `args` parameter is set to `null`.
+  HRESULT Invoke([in] ICoreWebView2Staging2* sender, [in] IUnknown* args);
```

Comment:
We've been doing ICoreWebView2* for event handler sender's args to make it possible to move to public without changing the interface